src/game_objects/player.cpp

```diff
@@ -60,6 +60,9 @@ void Player::update() {
 
     transform.set_position(glm::vec3((float)to_set.getX(), (float)to_set.getY(),
         (float)to_set.getZ()));
+
+    // Fires player position whenever player moves
+    events::dungeon::player_position_event(transform.get_position());
```

Comment:
Changing the event name to indicate that you're sending the position would be better. That way we know that it is getting the player position, not setting it.